services/ux-backend/main.go

```diff
@@ -48,9 +52,30 @@ func loadAndValidateServerConfig() (*serverConfig, error) {
 		return nil, err
 	}
 
+	config.namespace, err = readEnvVar("POD_NAMESPACE", "openshift-storage", func(str string) (string, error) {
+		return str, nil
+	})
+	if err != nil {
+		return nil, err
+	}
```

Comment:
Namespace does not make sense as a server config. It has no default value and is not user-configured. 
Not every piece of information coming from an Env var should be considered a server config.

Reply:
Totally agreed, I also felt this isn't necessarily a server param but we can argue the same for lifetime as that also belongs to a specific endpoint and so just used this config struct.

Reply:
> but we can argue the same for lifetime as that also belongs to a specific endpoint and so just used this config struct.

I don't agree. `tokenLifetimeInHours` is user-defined and has a default value. It does not matter if the config is effecting 1 endpoint, 5 endpoints or all endpoints. That is not the criteria.  

This about which type of information will go into a configmap when/if we decide to move the server config into a configmap

Reply:
> That is not the criteria.
- Yes, what I mentioned above was, I used a wrong criteria to arrive at reusing the server struct for storing namespace which I'll be removing in next refresh.

Reply:
done.